packages/hydrogen/src/customer/customer.ts

```diff
@@ -112,16 +112,18 @@ export type CustomerClient = {
 type CustomerClientOptions = {
   /** The client requires a session to persist the auth and refresh token. By default Hydrogen ships with cookie session storage, but you can use [another session storage](https://remix.run/docs/en/main/utils/sessions) implementation.  */
   session: HydrogenSession;
-  /** Unique UUID prefixed with `shp_` associated with the application, this should be visible in the customer account api settings in the Hydrogen admin channel. */
+  /** Unique UUID prefixed with `shp_` associated with the application, this should be visible in the customer account api settings in the Hydrogen admin channel. mock.shop doesn't automatically supply customerAccountId. Use h2 env pull to link your store credentials. */
   customerAccountId: string;
-  /** The account URL associated with the application, this should be visible in the customer account api settings in the Hydrogen admin channel. */
+  /** The account URL associated with the application, this should be visible in the customer account api settings in the Hydrogen admin channel. mock.shop doesn't automatically supply customerAccountUrl. Use h2 env pull to link your store credentials. */
   customerAccountUrl: string;
   /** Override the version of the API */
   customerApiVersion?: string;
   /** The object for the current Request. It should be provided by your platform. */
   request: CrossRuntimeRequest;
   /** The waitUntil function is used to keep the current request/response lifecycle alive even after a response has been sent. It should be provided by your platform. */
   waitUntil?: ExecutionContext['waitUntil'];
+  /** This is the url where the auth method should live in the loader. Can be a full url or start with / for relative url. Default is `/authorize` if not provided. */
```

Comment:
```suggestion
  /** This is the route in your app that authorizes the user after logging in. Make sure to call `customer.authorize()` within the loader on this route. It defaults to `/authorize`. */
```

If we decide to change the default, make sure to change this to say `/accounts/authorize`